src/script/lua_api/l_settings.cpp

```diff
@@ -26,12 +26,36 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "log.h"
 
 
+/* This protects:
+ * 'secure.*' settings from being set
+ * some mapgen settings from being set
+ *   (not security-criticial, just to avoid messing up user configs)
+ */
 #define SET_SECURITY_CHECK(L, name) \
```

Comment:
I suggest naming this `CHECK_SETTING_NAME` or `CHECK_SETTING_SECURITY`, as set is usually a verb. Same for the inline function

Reply:
Done.